crates/forge/src/main.rs

```diff
@@ -116,6 +125,9 @@ fn extract_failed_tests(tests_summaries: Vec<TestCrateSummary>) -> Vec<TestCaseS
 }
 
 fn test_workspace(args: TestArgs) -> Result<bool> {
+    if let ColorOption::Never = args.color {
+        env::set_var("CLICOLOR", "0");
+    }
```

Comment:
I think we should also set `CLICOLOR_FORCE != 0` when `ColorOption::Always` is used. We can then introduce a `match` statement here.

I think it will also allow us to test this feature since `CLICOLOR_FORCE != 0` enforces the colored output no matter what

Reply:
I agree that `always` should force the color output.

Maybe I should also add `auto` value to match `cargo test`:
```
--color when
    Control when colored output is used. Valid values:
        auto (default): Automatically detect if color support is available on the terminal.
        always: Always display colors.
        never: Never display colors.
```

What do you think about `auto` ?

Reply:
I think we can skip the `auto` option, since it basically does nothing

Reply:
> I think we can skip the `auto` option, since it basically does nothing

I think "do nothing" should be the default: as a user I expect that software by default should not change my colored output setting.

Here is my proposal:
```diff
-    if let ColorOption::Never = args.color {
-        env::set_var("CLICOLOR", "0");
+    match args.color {
+        ColorOption::Always => env::set_var("CLICOLOR_FORCE", "1"),
+        ColorOption::Never => env::set_var("CLICOLOR", "0"),
+        ColorOption::Auto => (),
     }
+

```

Reply:
I forgot we set `Always` as default, I thought the arg is `Option<ColorOption>`. I think it is fine to add `Auto` then to be consistent with Cargo, it makes the code a bit clearer too I think since wrapping with `Option` adds additional complexity. Good call imo!
cc @cptartur

Reply:
> Here is my proposal:
> 
> ```diff
> -    if let ColorOption::Never = args.color {
> -        env::set_var("CLICOLOR", "0");
> +    match args.color {
> +        ColorOption::Always => env::set_var("CLICOLOR_FORCE", "1"),
> +        ColorOption::Never => env::set_var("CLICOLOR", "0"),
> +        ColorOption::Auto => (),
>      }
> +
> ```

Looks reasonable to me, we can do that.

Reply:
`auto` option and test cases added.